domains/pente/src/test/java/io/kaleido/paladin/pente/domain/BondTest.java

```diff
@@ -221,15 +233,23 @@ void testBond() throws Exception {
             bondTracker.beginDistribution(bondCustodian, 1, 1);
 
             // Add Alice as an allowed investor
-            var investorRegistry = bondTracker.investorRegistry(bondCustodian);
-            investorRegistry.addInvestor(bondCustodian, aliceAddress);
+            var investorList = bondTracker.investorList(bondCustodian);
+            investorList.addInvestor(bondCustodian, aliceAddress);
+
+            // Create the atom factory on the base ledger
+            String atomFactoryAddress = testbed.getRpcClient().request("testbed_deployBytecode",
+                    "issuer",
+                    atomFactoryABI,
+                    atomFactoryBytecode,
+                    new HashMap<String, String>());
 
             // Alice deploys BondSubscription to the alice/custodian privacy group, to request subscription
             // TODO: if Alice deploys, how can custodian trust it's the correct logic?
             var bondSubscription = BondSubscriptionHelper.deploy(aliceCustodianInstance, alice, new HashMap<>() {{
```

Comment:
I found this a good reminder for me to google the initializer syntax, and I found some very good guidance that this anonymous-innerclass+initializer syntax is actually pretty nasty.

Fine for test, but not great for production code... and we're currently using it in both.

Instead we should move over to `Map.of(...` syntax: https://stackoverflow.com/a/59366753/21677265